packages/extension/src/newtab/ShortcutLinks.spec.tsx

```diff
@@ -1,8 +1,7 @@
-import React from 'react';
+import React, { act } from 'react';
```

Comment:
You removed act right?
Not sure is it used elsewhere in file?

Reply:
Ah, yes, but that is for the other file.